### ISSUE TITLE
isolinux: Wrong memtest path

### ISSUE THREAD

**Issue by PicoMitchell** [linuxmint/mint22-beta#45]
In the `/isolinux/live.cfg` file in the ISO, it seems like the `kernel /boot/memtest64.efi` line should be `kernel /boot/memtest64.bin` to load the correct file when booted into Legacy BIOS mode.

**Comment by clefebvre** [comment]
I remember using this one because the other one didn't work.. doesn't memtest64.efi work for you in BIOS mode?

**Comment by PicoMitchell** [comment]
Specifying `kernel /boot/memtest64.efi` does boot in Legacy BIOS, but I saw it throw errors on RAM that does not throw errors when booted into UEFI (or when running `memtest64.bin` from a Legacy BIOS iPXE boot). So while the program launches, I believe that it is not running properly since it's not running in the intended environment and could give false negatives.

With a little more testing, I also saw that `kernel /boot/memtest64.bin` fails to boot, but using `linux /boot/memtest64.bin` instead works (and does not throw the errors like `kernel /boot/memtest64.efi` did). From this documentation (https://www.syslinux.org/old/comboot.php) it looks like specifying `kernel` determines the boot method from the specified files extension and is likely attempting the wrong boot method because of the `.bin` extension while using `linux` forces the correct boot method.
